js/modules/networkgraph/networkgraph.src.js

```diff
@@ -531,13 +531,26 @@ seriesType('networkgraph', 'line', {
             }
         }
     },
+    /**
+     * Destroy all nodes (points), generatePoints() will take care of creating
+     * them again.
+     */
+    setData: function () {
+        if (this.nodes) {
+            this.nodes.forEach(function (node) {
+                node.destroy();
+            });
+            this.nodes.length = 0;
+        }
+
+        Series.prototype.setData.apply(this, arguments);
+    },
```

Comment:
We have this code in the sankey series too: https://github.com/highcharts/highcharts/blob/b5cdcdf80bdd7692afdf60ec3ecf3346ea73b649/js/modules/sankey.src.js#L443-L452

Shouldn't we move it to `mixins/node.js`?

Reply:
Thanks! Moved actually more methods:
- `generatePoints` (part of the code was the same)
- `setData` 
- `destroy`